src/hillmaker/hills.py

```diff
@@ -1,6 +1,18 @@
 """Hillmaker"""
 
-# Copyright 2022 Mark Isken
+# Copyright 2015, 2022 Mark Isken, Jacob Norman
```

Comment:
Take out the Apache stuff as I've moved hillmaker to an MIT License.

Reply:
Fixed in most recent commit